spec/request/service_route_bindings_spec.rb

```diff
@@ -1006,6 +1006,12 @@
                                                                  'code' => 10_015
                                                                }))
         end
+
+        it 'rolls back the transaction' do
+          api_call.call(admin_headers)
+
+          expect(service_instance.service_keys.count).to eq(0)
```

Comment:
Here you have to check `service_instance.routes`. Service keys are always 0 even without the transaction.

Reply:
Oh, good point